### ISSUE TITLE
na_ontap_user module - create snmpv3 user (usm)

### ISSUE THREAD

**Issue by xmax000** [ansible-collections/netapp#38]
##### SUMMARY
I would like to create an user for snmpv3 (usm), but the module do not support parameters like EngineID, authentication protocol, privacy protocol and remote-switch-ipaddress.
Could you add these parameters?
Thank you.

##### ISSUE TYPE
- Bug Report / Future Request

##### COMPONENT NAME
na_ontap_user module

##### ANSIBLE VERSION

```
ansible 2.9.7
```

**Comment by carchi8py** [comment]
I'll add an internal story to add support for snmpv3-login information.

**Comment by xmax000** [comment]
Thank you for adding support for snmpv3-login information.
I was able to successfully create a snmpv3 user with ansible.
however, I noticed that it is not possible to use the same username twice for different switches.

it works with the ontap cli:
security login create -user-or-group-name brocade -application snmp -authentication-method usm -role none -remote-switch-ipaddress 10.10.10.1
security login create -user-or-group-name brocade -application snmp -authentication-method usm -role none -remote-switch-ipaddress 10.10.10.2

https://docs.netapp.com/ontap-9/topic/com.netapp.doc.dot-mcc-inst-cnfg-fabric/GUID-DEED4D98-61CD-44DC-BA9E-EA548BD5A006.html?cp=11_0_5_8_6
